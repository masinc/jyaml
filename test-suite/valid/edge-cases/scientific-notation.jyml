# Scientific notation numbers
"small": 1.23e-10
"large": 9.87E+15
"negative": -4.56e-3