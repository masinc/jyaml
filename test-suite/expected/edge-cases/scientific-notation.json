{"small": 1.23e-10, "large": 9.87e+15, "negative": -0.00456}